src/python/pants/util/strutil.py

```diff
@@ -154,6 +154,16 @@ def hard_wrap(s: str, *, indent: int = 0, width: int = 96) -> Sequence[str]:
     ]
 
 
+def bullet_list(elements: Iterable[str]) -> str:
+    """Format the elements with `  * ` as a prefix and a newline separating each element.
```

Comment:
This says too much. The whole point is the caller need not care how the bullet list looks, just how to pad.